tests/reports.test.js

```diff
@@ -259,6 +276,10 @@ describe('reports', function ( ) {
        return true;
      };
 
+     window.setTimeout = function mockSetTimeout (call, delay) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["delay" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2306944046/issues/source?bid=2452379&fileBranchId=3191689#l279)